src/helix-ui/utils/index.js

```diff
@@ -55,6 +55,57 @@ export const KEYS = {
     Up: 38,
 };
 
+/**
+ * Defers execution of callback function until _next_ event loop.
+ *
+ * @param {Function} callback
+ */
+export function defer (cb) {
+    return () => setTimeout(cb, 0);
+}
+
+/**
+ * Generate a unique ID
+ *
+ * **Pseudo-random Algorithm**
+ * This functionality is pseudo-random, and you should not depend on 100%
+ * random values. Given a large enough dataset, this method has the
+ * potential to generate duplicate values.
+ *
+ * For the purposes of most applications, the dataset is small enough that
+ * the potential for duplicate values is almost 0, meaning that it's good
+ * enough for use.
```

Comment:
if possible, add a link to the SO for this.

Reply:
Couldn't find the original inspiration, but found a gist that is pretty darn close.